### PR TITLE
sign_in: clarify github OR user/pwd

### PR DESCRIPTION
**Pull request by mattscilipoti**
- move link out of form
- add image assets, for github image
- minor tweaks to login feature
- github images are from https://github.com/logos

### CODE REVIEW

File: spec/features/login_spec.rb [hunk 1]
```diff
@@ -1,17 +1,18 @@
 require 'rails_helper'
 
-describe "the login process", :type => :feature do
+RSpec.describe "the login process", :type => :feature do
   before :each do
     User.create(:username => 'testuser', :password => 'password')
   end
 
-  it "signs a user in" do
+  scenario "sign in via username & password" do
```
Comment (RobertAKARobin) [inline spec/features/login_spec.rb]: Didn't know this was a thing!

**Comment by RobertAKARobin** [comment]
:+1:

**Comment by RobertAKARobin** [comment]
Wanna take a look at mine? https://github.com/ga-dc/garnet/pull/269

**Comment by jshawl** [comment]
:+1:

**Pull request merged**